### DISCUSSION TITLE
Timeline 'file saved'

### DISCUSSION THREAD

**Discussion by Harain** [gitkraken/vscode-gitlens#1982] [Q&A] [answered]
hi, just installed this and have something that's bothering me

any way to remove all the file saved entries in timeline?
![image](https://user-images.githubusercontent.com/6484567/165754005-56a4332c-9404-430c-835a-5287f05f46fd.png)

**Comment by pelukron** [comment]
> hi, just installed this and have something that's bothering me
> 
> any way to remove all the file saved entries in timeline?
> ![image](https://user-images.githubusercontent.com/6484567/165754005-56a4332c-9404-430c-835a-5287f05f46fd.png)
> 

Is a new feature about history of local. Save in vscode, idk which was the release lemme check

**Reply by pelukron** [reply]
https://code.visualstudio.com/updates/v1_66#_workbench

**Comment by eamodio** [comment, marked as answer]
The Timeline view and that new local history feature is provided by VS Code not GitLens. You can toggle the new local history off using the funnel icon on the Timeline view header and unchecking Local History.
